code/game/gamemodes/objective.dm

```diff
@@ -121,7 +121,7 @@
 	return target
 
 /datum/objective/assassinate/check_completion()
-	return !target || !considered_alive(target)
+	return !target || !considered_alive(target) || considered_afk(target)
```

Comment:
the first `!target` can be removed, it's checked in both `considered_alive` and `considered_afk`

Reply:
That was already there, but if you REALLY want ME to remove it, I can do that.

Reply:
do it

Reply:
yes, please remove it

Reply:
@Y0SH1M4S73R Please don't take this personal. Whenever we touch code we usually clean up at least a little bit even if we didn't write the messy code, and thus we also require others to do the same if we see it.